src/extension.ts

```diff
@@ -36,7 +36,7 @@ export async function activate() {
             provider.gatherCode(cell, false);
           }
         } else {
-          vscode.window.showInformationMessage(localize.Common.runCells());
+          vscode.window.showInformationMessage(localize.Common.runCells() + ' ' + localize.Common.reopenNotebooks());
```

Comment:
This seems ok, so approving. If you do have an easy way to detect the "just installed" state might be nice to specifically test for that. Otherwise you have a much wordier error message here when most of the time the second part of the message (reopenNotebooks) doesn't apply.

Reply:
https://github.com/microsoft/vscode-gather/issues/41